src/App.vue

```diff
@@ -573,16 +574,21 @@ export default {
       }
     },
 
-    /** Updates Launch Darkly with current user info. */
+    /**
+     * Updates Launch Darkly with current user info.
+     * Set the full name in the Authentication store.
+     */
     async updateLaunchDarkly (userInfo: any): Promise<void> {
       // since username is unique, use it as the user key
       const key: string = userInfo.username
       const email: string = userInfo.contacts[0]?.email || userInfo.email
       const firstName: string = userInfo?.firstname
       const lastName: string = userInfo?.lastname
+      const fullName: string = firstName + ' ' + lastName
       // store Keycloak roles in custom object
       const custom = { roles: this.getKeycloakRoles } as any
 
+      this.setUserFullName(fullName)
```

Comment:
Don't do this here. It's a "side effect" -- not the purpose of this function; no one would look here to see where user full name is set.

Also, I think you should save the entire user info object into store -- we've already done the work of fetching it; there might be other info in there we could use later.

Reply:
I'm resolving this conversation because the initial issue is fixed.

However, you have created another issue, but it's captured in another conversation so it's still OK to resolve this one.